cabal.project

```diff
@@ -108,3 +108,39 @@ package snap-server
 -- IMPORTANT
 -- Do NOT add more source-repository-package stanzas here unless they are strictly
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
+
```

Comment:
Are these things not yet in CHaP?

Is it worth waiting until they are?

Reply:
I should probably not have been looking at each commit individually. However, I do also think that the commit history should be cleaned up so that it looks like those SRPs were never added.